kedro-org/kedro-viz
# Skip optional plugins while running Kedro Viz

## Description

Resolves #1159 


## Development notes

* Add additional option during kedro viz run to ignore plugins
* Modify kedro-dataset-stats hook name
* Refactor public method names to private

## QA notes
1. Install local kedro Viz package by executing `pip install -e package`
2. Navigate to demo project `cd demo-project`
3. Install kedro mlflow `pip install kedro-mlflow`
4. Run kedro viz without ignoring plugins - `kedro viz`
5. You will find logs in the terminal related to mlflow plugin
6. Re-run kedro viz with --ignore-plugins flag - `kedro viz --ignore-plugins`
7. There should not be any logs related to mlflow.

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1544"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
